tests/events/microsoft/test_parse.py

```diff
@@ -1380,6 +1380,16 @@ def test_parse_event_singular():
     assert event.read_only is False
 
 
+def test_parse_event_with_invalid_organizer():
+    event_with_invalid_organizer = single_instance_event.copy()
+    event_with_invalid_organizer["organizer"] = {
+        "name": "Garrett’s Bespoke Barber Shop",
```

Comment:
Nit: Could we make this field non-unicode to show that the `address` field is actually causing the error? Or maybe this could be a parametrized test?